src/components/LanguageEngagementListItemCard/LanguageEngagementListItem.graphql

```diff
@@ -9,13 +9,15 @@ fragment LanguageEngagementListItem on LanguageEngagement {
       displayName {
         value
       }
-      ethnologuePopulation {
-        value
+      ethnologue {
+        population {
+          value
+        }
```

Comment:
Not necessary at this view. `population` is the calculated value of either `populationOverride` or `ethnologue.pupulation`.

Reply:
Changed